specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Run notebook parameters",
+          "additionalProperties": {
+            "$ref": "#/definitions/RunNotebookParameter"
+          }
+        },
+        "notebookContent": {
+          "$ref": "../2021-06-01-preview/notebooks.json#/definitions/NotebookResource"
+        }
+      },
+      "required": [
+        "id",
+        "notebook"
+      ]
+    },
+    "RunNotebookSnapshotResult": {
+      "description": "Run notebook snapshot result.",
+      "type": "object",
+      "properties": {
+        "snapshot": {
+          "$ref": "#/definitions/RunNotebookSnapshot"
+        },
+        "error": {
+          "$ref": "#/definitions/RunNotebookError"
+        },
+        "runId": {
+          "type": "string",
+          "description": "Run id."
+        },
+        "runStatus": {
+          "description": "Status of the run notebook.",
+          "type": "string"
+        },
+        "lastCheckedOn": {
+          "description": "Timestamp of last update.",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "runId",
+        "runStatus",
+        "snapshot"
+      ]
+    },
+    "RunNotebookSnapshotResponse": {
+      "description": "Run notebook snapshot response.",
+      "type": "object",
+      "properties": {
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Response message."
+        },
+        "result": {
+          "$ref": "#/definitions/RunNotebookSnapshotResult"
+        }
+      }
+    },
+    "RunNotebookSparkSessionOptions": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "The session kind"
+        },
+        "proxyUser": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "User to impersonate when starting the session"
+        },
+        "name": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "The name of this session"
+        },
+        "jars": {
+          "uniqueItems": false,
+          "x-nullable": true,
+          "type": "array",
+          "description": "jars to be used in this session",
+          "items": {
+            "type": "string"
+          }
+        },
+        "pyFiles": {
+          "x-ms-client-name": "pythonFiles",
```

Comment:
Is it too late to change this name in the API directly?  That would be much preferred.

Reply:
We cannot change the property name because it is open source LIVY standard. https://livy.incubator.apache.org/docs/latest/rest-api.html